ringpop.go

```diff
@@ -330,11 +331,10 @@ func (rp *Ringpop) setState(s state) {
 // Bootstrap starts communication for this Ringpop instance.
 //
 // When Bootstrap is called, this Ringpop instance will attempt to contact
-// other instances from a seed list provided either in the BootstrapOptions or
-// as a JSON file.
+// other instances from the DiscoverProvider.
 //
 // If no seed hosts are provided, a single-node cluster will be created.
-func (rp *Ringpop) Bootstrap(userBootstrapOpts *swim.BootstrapOptions) ([]string, error) {
+func (rp *Ringpop) Bootstrap(userbootstrapOpts *swim.BootstrapOptions) ([]string, error) {
```

Comment:
I refactored the whole section, `userbootstrapOpts` and the likes are gone from this file.